rotkehlchen/tests/api/test_makerdao_dsr.py

```diff
@@ -144,14 +144,14 @@ def mock_etherscan_for_dsr(
     proxy1_contents = proxy1[2:].lower()
     proxy2_contents = proxy2[2:].lower()
 
-    def mock_requests_get(url: str, *args: Any, **kwargs) -> _patch | MockResponse:
-        if 'etherscan.io/api?module=proxy&action=eth_blockNumber' in url:
+    dsr_params = params
+
+    def mock_requests_get(url: str, params: dict[str, str], *args: Any, **kwargs) -> _patch | MockResponse:  # noqa: E501
+        if 'etherscan.io/api' in url and {'module': 'proxy', 'action': 'eth_blockNumber'}.items() <= params.items():  # noqa: E501
```

Comment:
minor suggestion here would be to add this in a constant in the same file to not repeat

```suggestion
        if ETHERSCAN_API in url and {'module': 'proxy', 'action': 'eth_blockNumber'}.items() <= params.items():  # noqa: E501
```

Reply:
I actually put the constant in tests/utils/bockchain.py since its also used there and seems like a better place for it to be defined than in the makerdao test.